.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
 	root: true,
 	'extends': [
-		'wpcalypso/react',
+		'wpcalypso/react-a11y',
```

Comment:
I think we should go with the `plugin:jsx-a11y/recommended` ruleset, rather than defining them in `wpcalypso/react-a11y` – whether we make that change here, or change [eslint-config-wpcalypso](https://github.com/Automattic/eslint-config-wpcalypso/blob/master/react-a11y/index.js), I don't have a preference.

My reasoning is that the rules have updated since we set up `wpcalypso/react-a11y`, and it would be easier to use the recommendations, rather than go through and update our rules list.

Reply:
> My reasoning is that the rules have updated since we set up `wpcalypso/react-a11y`, and it would be easier to use the recommendations, rather than go through and update our rules list.

Makes perfect sense, esp. from a maintainability and (pseudo-)standards conformance (comparable to widespread adoption of `prettier`) POV 👍 

> whether we make that change here, or change [eslint-config-wpcalypso](https://github.com/Automattic/eslint-config-wpcalypso/blob/master/react-a11y/index.js), I don't have a preference.

Hmm, tough call -- part of me wants to do it as far upstream as possible (i.e. in `eslint-config-wpcalypso`), but OTOH, if we'd be essentially just aliasing `plugin:jsx-a11y/recommended` there, it'd make sense to view _that_ as our upstream and de-couple `eslint-config-wpcalypso` from it. So I might do it here, and maybe even file a PR against `eslint-config-wpcalypso` to drop the `react-a11y` config there altogether(?)

Reply:
> file a PR against `eslint-config-wpcalypso` to drop the `react-a11y` config there altogether(?)

https://github.com/Automattic/eslint-config-wpcalypso/pull/15